src/main/groovy/edu/oregonstate/mist/positions/resources/PositionsResource.groovy

```diff
@@ -0,0 +1,43 @@
+package edu.oregonstate.mist.positions.resources
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.api.jsonapi.ResultObject
+import edu.oregonstate.mist.positions.db.PositionDAO
+import groovy.transform.TypeChecked
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.annotation.security.PermitAll
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+import javax.ws.rs.QueryParam
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+@Path("positions")
+@Produces(MediaType.APPLICATION_JSON)
+@PermitAll
+@TypeChecked
+class PositionsResource extends Resource {
+    Logger logger = LoggerFactory.getLogger(PositionsResource.class)
+    private PositionDAO positionDAO
+
+    PositionsResource(PositionDAO positionDAO) {
+        this.positionDAO = positionDAO
+    }
+
+    @Timed
+    @GET
+    Response getPositions(@QueryParam('businessCenter') String businessCenter) {
```

Comment:
The "type" query parameter should be added as a required parameter as it says in the Swagger. It should be checked for the value "student" as well.